app/models/automation_task.rb

```diff
@@ -1,6 +1,6 @@
 class AutomationTask < MiqRequestTask
   alias_attribute :automation_request, :miq_request
-
+  has_one :container_deployment
```

Comment:
Not sure whether this belongs here (probably not). @blomquisg @gmcculloug @Fryguy ?